### DISCUSSION TITLE
Recent SQL script behavior (CE)

### DISCUSSION THREAD

**Discussion by tempest666** [dbeaver/dbeaver#17455] [Q&A] [answered]
Was "Recent SQL script" behavior changed from global version 21 to 22?
In version 21 when i clicked Recent SQL script it opened most recent script for that specific connection, 
but now in version 22 it opens most recent script globally from any connection.

**Comment by LonwoLonwo** [comment]
Hello @tempest666 

What exact version of DBeaver do you use?

**Reply by tempest666** [reply]
@LonwoLonwo 
Currently i'm using 22.1.4 CE

**Comment by tempest666** [comment, marked as answer]
I see the problem is resolved in the version 22.1.5
